infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/AbstractWorkspaceServiceAccount.java

```diff
@@ -127,19 +131,31 @@ private void ensureImplicitRolesWithBindings(Client k8sClient) {
         k8sClient,
         METRICS_ROLE_NAME,
         Arrays.asList("pods", "nodes"),
+        null,
         singletonList("metrics.k8s.io"),
         Arrays.asList("list", "get", "watch"),
         serviceAccountName + "-metrics");
+
+    // credentials-secret role
+    ensureRoleWithBinding(
+        k8sClient,
+        SECRETS_ROLE_NAME,
+        singletonList("secrets"),
+        singletonList("che-credentials-secret"),
```

Comment:
Can we choose a Che-agnostic name -- e.g. `workspace-credentials-secret`? If we want to do a similar thing in DWO, it's a lot easier to justify if it's not labelled with a specific platform.

Reply:
+1 for `workspace-credentials-secret`

Reply:
\+ please consider moving that secret name somewhere to constants

Reply:
done